classes/profile_table.php

```diff
@@ -146,4 +155,21 @@ public function col_user(object $record): string {
             }
         }
     }
+
+    /**
+     * Display for the delete column
+     *
+     * @param object $record
+     * @return mixed
+     * @throws \coding_exception
+     * @throws \moodle_exception
+     */
+    public function col_delete(object $record) {
```

Comment:
Not at action for now, just more of a heads up there is a defacto pattern in moodle to have an 'action' column and throw a bunch of icons like delete, details, up, down etc all in the one place. We can refactor this later when we add more actions